finetune/encoding/target_encoders.py

```diff
@@ -66,6 +66,21 @@ def transform(self, y):
         labels = super().transform(y)
         return self._make_one_hot(labels)
 
+    def inverse_transform(self, one_hot):
+        if len(one_hot.shape) == 2:
```

Comment:
What does it normally equal? Is it not always `[batch, num_classes]`?

Reply:
So... the inverse_transform() call of this function is also unfortunately used to convert from class ids to class names in predict() right now. Which violates the assertion that `x = inverse_transform(transform(x))`

Reply:
Maybe the best solution to this is to ensure that the output of the predict() method of any model type that uses OneHotLabelEncoder as it's target encoder is one hot encoded